encoding/thrift/thriftrw-plugin-yarpc/main.go

```diff
@@ -25,24 +25,42 @@
 package main
 
 import (
+	"flag"
 	"path/filepath"
 	"strings"
 
 	"go.uber.org/thriftrw/plugin"
 	"go.uber.org/thriftrw/plugin/api"
 )
 
+var (
+	_context             = flag.String("context", "context", "generate code with the provided context")
+	_unaryHandlerWrapper = flag.String("unary-handler-wrapper",
+		"go.uber.org/yarpc/encoding/thrift.UnaryHandlerFunc",
+		"wrapper for import path where unary handler functions or wrappers are implemented")
+	_onewayHandlerWrapper = flag.String("oneway-handler-wrapper",
+		"go.uber.org/yarpc/encoding/thrift.OnewayHandlerFunc",
+		"wrapper import path where oneway handler functions or wrappers are implemented")
+)
+
+type userContext struct {
```

Comment:
This doesn't need to be a separate type. You can stick all these fields on the anonymous `templateData` struct.